packages/lesswrong/lib/collections/curationNotices/collection.ts

```diff
@@ -0,0 +1,28 @@
+import schema from './schema';
+import { createCollection } from '../../vulcan-lib';
+import { addUniversalFields, getDefaultResolvers } from '../../collectionUtils'
+import { getDefaultMutations } from '../../vulcan-core/default_mutations';
+import { makeEditable } from "../../editor/make_editable";
+
+export const CurationNotices: CurationNoticesCollection = createCollection({
+  collectionName: 'CurationNotices',
+  typeName: 'CurationNotice',
+  schema,
+  resolvers: getDefaultResolvers('CurationNotices'),
+  mutations: getDefaultMutations('CurationNotices'),
```

Comment:
As noted [here](https://github.com/ForumMagnum/ForumMagnum/pull/9704/files#r1742730065), this should have permissions checks on the mutations (and also a `CurationNotices.checkAccess` function at the top-level)

Reply:
done